conf/st2.conf.sample

```diff
@@ -100,6 +100,16 @@ logging = conf/logging.exporter.conf
 # Directory to dump data to.
 dump_dir = /opt/stackstorm/exports/
 
+[garbagecollector]
```

Comment:
I was thinking about it, but imo it's an overkill (and it makes monitoring this service harder and more complex)

If no TTLs are configured, the service is very light on resources (it only polls every 300 seconds) so it's not a big deal if it's running.